### PR TITLE
Ins 2420 logger fix skip call number

### PR DESCRIPTION
**Pull request by nordicdyno**

### CODE REVIEW

File: instrumentation/inslogger/inslogger.go [hunk 1]
```diff
@@ -78,11 +78,16 @@ func ContextWithTrace(ctx context.Context, traceid string) context.Context {
 }
 
 func getLogger(ctx context.Context) insolar.Logger {
-	l := ctx.Value(loggerKey{})
-	if l == nil {
-		return logger.GlobalLogger
+	val := ctx.Value(loggerKey{})
+	if val == nil {
+		return logger.GlobalLogger.ChangeSkipFrameCount(1).WithField("loginstance", "global_inslog")
```
Comment (ruz) [inline instrumentation/inslogger/inslogger.go]: this "loginstance" field is for debugging and should be removed.

File: insolar/logger.go [hunk 2]
```diff
@@ -86,8 +86,8 @@ type Logger interface {
 
 	// WithCaller switch on/off 'caller' field computation.
 	WithCaller(flag bool) Logger
-	// WithSkipFrameCount configures skipFrameCount for 'caller' field computation.
-	WithSkipFrameCount(skipFrameCount int) Logger
+	// ChangeSkipFrameCount changes skipFrameCount by delta value.
+	ChangeSkipFrameCount(delta int) Logger
```
Comment (ruz) [inline insolar/logger.go]: I don't think this should be renamed. I understand that this method adds or removes delta and it's ok. I prefer "With" prefix, it's more clear that it is a modifier.

Comment (ruz) [reply insolar/logger.go]: May be better name would be "WithSkipFrameDelta". What do you think?

Comment (nordicdyno) [reply insolar/logger.go]: 'delta' is already in the function signature, so my solution is just to stop on WithSkipFrameCount with improved godoc description

**Pull request merged**